lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -25,24 +26,10 @@ def initialize(payload)
       @payload = payload
     end
 
-    def description
```

Comment:
I think removing this is the right thing to do, but it seems that policy pages don't have a description set, which means they won't display anything on results pages after this change. Example: https://www.gov.uk/api/content/government/policies/academies-and-free-schools

This also affects external search - eg googling brexit gives you

> Brexit. From: Department for Exiting the European Union. The UK voted to leave the European Union in the 23 June 2016 EU referendum. The government is now preparing to leave the EU in the best possible way for the UK's national interest. Subscribe to email alerts. Search. Document type. Case study

Instead of just 

> The UK voted to leave the European Union in the 23 June 2016 EU referendum. The government is now preparing to leave the EU in the best possible way for the UK’s national interest.

Before we merge this PR, we should make a change to policy publisher to actually populate descriptions.